File: C/URI_judge_1018.c
#include <stdio.h>

int main(){
    int n,c;

    scanf("%d",&n);
    printf("%d\n",n);

    printf("%d nota(s) de R$ 100,00\n", n/100);
    c = (n%100);

    printf("%d nota(s) de R$ 50,00\n", c/50);
    c = (c%50); 

    printf("%d nota(s) de R$ 20,00\n", c/20);
    c = (c%20);

    printf("%d nota(s) de R$ 10,00\n", c/10);
    c = (c%10);

    printf("%d nota(s) de R$ 5,00\n", c/5);
    c = (c%5);
    
    printf("%d nota(s) de R$ 2,00\n", c/2);
    c = (c%2);

    printf("%d nota(s) de R$ 1,00\n", c/1);
}

File: php/URI_Judge_1018.php
<?php

    //Notas  em cédulas.
    $notas = array(100,50,20,10,5,2,1);

    //Adicinando valor.
    $valor = readline();

    echo "$v\n";

    //Variavel contador para os 7 resultados.
    for ($x = 0; $x < 7; $x++) {
        
        //variavel auxiliar
        $y = $valor / $notas[$i];

        echo (int) $y . " nota(s) de R$ " . $notas[$x] . ",00\n";

        //notas = notas resto 100,50...
        $valor %= $notas[$x];
    }
?>
